sycl/doc/extensions/template.asciidoc

```diff
@@ -109,6 +111,19 @@ Be aware that the compiler is not able to issue a diagnostic to warn you if
 this happens.
 ====
 
+_Use this if the extension is not supported on at least one backend :_
+
+It is(n't) possible to use this extension to write fully portable code.
+Here "fully portable" means that a program can compile and execute using
+any backend supported by DPC++.
```

Comment:
Isn't this the same case as the NOTE we already have?  I think the template should propose only one paragraph for the case when the extension is supported only on some backends.  Can you combine your text with the existing NOTE and/or remove one of them?  Since we are adding a new section, we probably don't need the NOTE callout anymore, and we can just use a normal paragraph.